src/include/units/quantity.h

```diff
@@ -32,6 +32,7 @@
 #endif
 
 #include <ostream>
+#include <iostream>
```

Comment:
Probably we do not need this header her?

Reply:
ooops .. fair enough... detritus from debugging, I suspect.